devbox.lock

```diff
@@ -1121,7 +1121,18 @@
     },
     "procps": {
       "resolved": "github:NixOS/nixpkgs/3afd19146cac33ed242fc0fc87481c67c758a59e?narHash=sha256-OxYwCGJf9VJ2KnUO%2Bw%2FhVJVTjOgscdDg%2FlPv8Eus07Y%3D#procps",
-      "source": "nixpkg"
+      "source": "nixpkg",
+      "systems": {
+        "aarch64-darwin": {
```

Comment:
This looks macOS specific right?

It's odd that there are so many changes to the file... essentially I ran:
```
devbox update go-task
```

... and this is the result.